components/python/import-profiler/import-profiler.license

```diff
@@ -0,0 +1,4 @@
+# Resources:
```

Comment:
put here the text of license, either from the module or from one of the locations.

Reply:
No need in links here.